docs/source-pytorch/accelerators/mps_basic.rst

```diff
@@ -0,0 +1,48 @@
+:orphan:
+
+.. _mps_basic:
+
+MPS training (basic)
+====================
+**Audience:** Users looking to train on their Apple silicon GPUs.
+
+.. warning::
+
+   The MPS accelerator as well as the PyTorch backend are still very experimental.
+   So far not all operations are supported, but more ops are coming every day due to development from the PyTorch Team.
+   You can use ``PYTORCH_ENABLE_MPS_FALLBACK=1 python your_script.py`` to fall back to cpu for unsupported operations.
+
+
+----
+
+What is Apple silicon?
+----------------------
+Apple silicon chips are a unified system on a chip (SoC) developed by Apple based on the ARM design.
+Among other things, they feature a CPU-cores, GPU-cores, a neural engine and shared memory between all of those.
+
+----
+
+So it's a CPU?
+--------------
+Among other things yes, it includes CPU-cores. However, when running on the ``CPUAccelerator``, not the full potential of hardware acceleration the M-Socs are capable of, is used because they also feature a GPU and a neural engine.
```

Comment:
```suggestion
Apple silicon includes CPU-cores among several other features. However, the full potential for the hardware acceleration of which the M-Socs are capable is unavailable when running on the ``CPUAccelerator``. This is because they also feature a GPU and a neural engine.
```